File: src/string/checkCapitalized/test_check_capitalized.py
from check_capitalized import *
import pytest


@pytest.mark.parametrize('input, expected', [
    ('chris alan', False),
    ('Chris alan', False),
    ('chris Alan', False),
    ('Chris Alan', True)
])
def test_check_capitalized(input, expected):
    assert check_capitalized(input) == expected

File: src/string/checkCapitalized/TestCheckCapitalized.java
package string.checkCapitalized;

import static org.junit.Assert.assertEquals;
import static string.checkCapitalized.CheckCapitalized.checkCapitalized;

import org.junit.jupiter.params.ParameterizedTest;
import org.junit.jupiter.params.provider.CsvSource;

public class TestCheckCapitalized {

    @ParameterizedTest
    @CsvSource(value = {
            "chris alan, false",
            "Chris Alan, true",
            "chris Alan, false",
            "Chris alan, false",
    })
    void testCheckCapitalized(String input, boolean expected) {
        assertEquals(expected, checkCapitalized(input));
    }
}